core/src/main/scala/quasar/fs/PathError.scala

```diff
@@ -29,11 +29,11 @@ sealed trait PathError2 {
 }
 
 object PathError2 {
-  private final case class PathExists0(path: AbsPath[Sandboxed])
+  final case class PathExists0(path: AbsPath[Sandboxed])
```

Comment:
So I need to look into what a Prism is. I don't like `fold` as I find it to be less readable.

Is there a specific issue with making them public?